.github/workflows/main.yml

```diff
@@ -0,0 +1,735 @@
+        run: haxe RunCi.hxml
+        working-directory: ${{github.workspace}}/tests
+        env:
+          SAUCE_ACCESS_KEY: matrix.SAUCE_ACCESS_KEY
+      
+
+  mac-test:
+    needs: mac-build
+    runs-on: macos-latest
+    env:
+      PLATFORM: mac
+      TEST: ${{matrix.target}}
+      HXCPP_COMPILE_CACHE: ~/hxcache
+    strategy:
+      fail-fast: false
+      matrix:
+        target: [macro, js, hl, cpp, 'java,jvm', cs, php, python, lua, flash9, neko]
+        include:
+          - target: hl
+            BREW_PACKAGES: ninja
+    steps:
+      - uses: actions/checkout@master
+        with:
+          submodules: recursive
+      - uses: actions/download-artifact@v1
+        with:
+          name: macBinaries
+
+      - name: Install Neko using snapshot from S3 (Windows)
+        if: startsWith(env.PLATFORM, 'windows')
+        shell: pwsh
+        run: |
+          $DOWNLOADDIR="./temp"
+          new-item -Name $DOWNLOADDIR -ItemType directory
+          Invoke-WebRequest https://build.haxe.org/builds/neko/$env:PLATFORM/neko_latest.zip -OutFile $DOWNLOADDIR/neko_latest.zip
+          Expand-Archive $DOWNLOADDIR/neko_latest.zip -DestinationPath $DOWNLOADDIR
+          $NEKOPATH = Get-ChildItem $DOWNLOADDIR/neko-*-*
+          Write-Host "::add-path::$NEKOPATH"
+          Write-Host "::set-env name=NEKOPATH::$NEKOPATH"
+      
+      - name: Install Neko using snapshot from S3 (Unix)
+        if: (startsWith(env.PLATFORM, 'linux'))
+        run: |
+          set -ex
+          sudo add-apt-repository ppa:haxe/snapshots -y
+          sudo apt-get update -qqy
+          sudo apt-get install -qqy neko
+      
+      - name: Install Neko using snapshot from S3 (Mac)
+        if: (startsWith(env.PLATFORM, 'mac'))
+        run: |
+          set -ex
+          HOMEBREW_NO_AUTO_UPDATE=1
+          brew install neko
+      
+      - name: Print Neko version
+        run: neko -version 2>&1
+      
+      
+      - name: Setup Haxe
+        run: |
+          # mkdir ./macBinaries
+          # curl -sSL https://build.haxe.org/builds/haxe/mac/haxe_latest.tar.gz -o ./macBinaries/haxe_bin.tar.gz
+      
+          set -ex
+          tar -xf macBinaries/*_bin.tar.gz -C macBinaries --strip-components=1
+          sudo mkdir -p /usr/local/bin/
+          sudo mkdir -p /usr/local/share/haxe/
+          sudo ln -s `pwd`/macBinaries/haxe /usr/local/bin/haxe
+          sudo ln -s `pwd`/macBinaries/haxelib /usr/local/bin/haxelib
+          sudo ln -s `pwd`/macBinaries/std /usr/local/share/haxe/std
+      
+      - name: Print Haxe version
+        run: haxe -version
+      
+      - name: Setup haxelib
+        run: |
+          set -ex
+          mkdir ~/haxelib
+          haxelib setup ~/haxelib
+      
+      - name: Install homebrew packages
+        if: matrix.BREW_PACKAGES
+        run: brew install ${{matrix.BREW_PACKAGES}}
+      
+      - name: Test
+        run: |
+          # LC_CTYPE=C
+          # LANG=C
+          haxe RunCi.hxml
```

Comment:
I commented on this in #9262, you need to add a line to disable tests which are invalid on Mac:

https://github.com/HaxeFoundation/haxe/blob/2b6424c1a21cc4d19474ef51dc8eac66b9ce220f/azure-pipelines.yml#L178-L180

Reply:
Thanks for hint, head post updated